linode_api4/login_client.py

```diff
@@ -490,9 +490,10 @@ def refresh_oauth_token(self, refresh_token):
 
     def expire_token(self, token):
```

Comment:
We might want to add a `revoke_token` method alias here to match the naming of the non-legacy endpoint.